ansible/templates/ubuntu2204/omero-web-init.d.j2

```diff
@@ -0,0 +1,76 @@
+{% if doc %}
+Should you wish to run OMERO.web automatically, a `init.d` file could be created. See below an example file `omero-web-init.d`::
+
+{% endif %}
+#!/bin/bash
+#
+# /etc/init.d/omero-web
+# Subsystem file for "omero" web
+#
+### BEGIN INIT INFO
+# Provides:             omero-web
+# Required-Start:       $local_fs $remote_fs $network $time omero postgresql
```

Comment:
this must be copied from before since postgresql should not be required here

Reply:
All the `ubuntu` versions and `debian` have the same lines here.
To remove `postgres` do I remove that line (and the following lines?) completely, or just remove the `postgresql ` from that line and the next?